.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
```

Comment:
enabling the cache may lead to subtle side effects (and does not speed up things significantly imo)
I'd recommend disabling it